src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.CoreRT.cs

```diff
@@ -43,7 +43,7 @@ public void StartProfileOptimization(string profile)
 
         private Assembly InternalLoadFromPath(string? assemblyPath, string? nativeImagePath)
         {
-            throw new PlatformNotSupportedException();
+            return ReflectionAugments.ReflectionCoreCallbacks.Load(assemblyPath);
```

Comment:
This is not passing down the AssemblyLoadContext, so it won't actually work properly when multiple assemblies with the same identity get loaded. This change is just following the existing pattern for `byte[]` so it is not introducing a brand new problem. Maybe add TODO comment here to make it obvious that this is not 100% correct implementation?